vnext/Shared/JSI/ChakraRuntime.cpp

```diff
@@ -16,14 +16,10 @@
 #include <sstream>
 #include <unordered_set>
 
-#ifdef CHAKRACORE
-#include <ChakraCore.h>
-#else
 #ifndef USE_EDGEMODE_JSRT
```

Comment:
Curious if the define of `USE_EDGEMODE_JSRT` has value if this always sets it anyway and we have it defined in all vcxproj files as well.

Reply:
Probably not. I applied the minimum amount of changes possible until failures (unit tests) are fixed.

Then I'll shrink this code as much as possible.